### DISCUSSION TITLE
Script for SVD

### DISCUSSION THREAD

**Discussion by lesani-ali** [chulminy/CIVE497-CIVE700#158] [General]
Hi Dr. Yeum, 

During your lecture on Singular Value Decomposition (SVD), you provided us with a MATLAB script for SVD computations. Could you kindly share that script with us?

**Comment by chulminy** [comment]
Good question.

Please see here: https://github.com/chulminy/CIVE497-CIVE700/tree/master/notebook/projective_geometry

Chul Min
